src/chain/store/chain_store.rs

```diff
@@ -194,11 +196,15 @@ where
 
     /// Returns Tipset from key-value store from provided CIDs
     #[tracing::instrument(skip_all)]
-    pub fn tipset_from_keys(&self, tsk: &TipsetKeys) -> Result<Arc<Tipset>, Error> {
+    pub fn tipset_from_keys(&self, tsk: &TipsetKeys) -> Result<Option<Arc<Tipset>>, Error> {
```

Comment:
`Tipset` has `Tipset::load` and `Tipset::load_required`. How about we mirror that naming convention and add `ChainStore::load_tipset`, `ChainStore::load_required_tipset`, `ChainIndex::load_tipset`, and `ChainIndex::load_required_tipset`?

Reply:
Fixed.